config.json

```diff
@@ -180,6 +197,18 @@
                     "platforms": ["Android"]
                 }
             ]
+        },
+        {
+            "name": "Frequently Asked Questions",
+            "platforms": ["Android","JavaSE","JavaEE"],
```

Comment:
Both of these are Android only right now.

Reply:
removed